models/hls.go

```diff
@@ -38,20 +38,35 @@ type LiveHLSManifest struct {
 	TimeShift  *float64 `json:"timeShift,omitempty"`
 }
 
+type HLSAudioGroupDefinition struct {
+	Name     string `json:"name,omitempty"`
```

Comment:
If this is optional I think this should be a `*string`

Reply:
`Name` is mandatory.